### PR TITLE
[chromecast-caf-receiver] PlayerManger.getStats()

### PR DESCRIPTION
**Pull request by tomhaines432**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [X] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://developers.google.com/cast/docs/reference/web_receiver/cast.framework.PlayerManager#getStats>>
<<https://developers.google.com/cast/docs/reference/web_receiver/cast.framework.Stats>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/chromecast-caf-receiver/cast.framework.d.ts [hunk 1]
```diff
@@ -1261,3 +1266,15 @@ export class AudioTracksManager {
     setActiveById(id: number): void;
     setActiveByLanguage(language: string): void;
 }
+
+/** Represents playback statistics */
+export class Stats {
```
Comment (Serabe) [inline types/chromecast-caf-receiver/cast.framework.d.ts:1271]: Can you copy and paste the documentation for this class and each property, please?

**Review by Serabe — CHANGES_REQUESTED** [review]

**Comment by Serabe** [comment]
Thank you! Just add some docs and it'll be fine to go.

File: types/chromecast-caf-receiver/cast.framework.d.ts (hunk 1 above)
Comment (brandonrisell) [reply types/chromecast-caf-receiver/cast.framework.d.ts:1271]: They have a link to the docs in the description, or do you mean as a comment in the source itself?

Comment (Serabe) [reply types/chromecast-caf-receiver/cast.framework.d.ts:1271]: In the code itself. That will show it in the user's IDE autocompletion.

**Pull request closed without merging**